### PR TITLE
Fix bug public key

### PR DESCRIPTION
**Pull request by svenvanderburg**

### CODE REVIEW

File: nanopub/client.py [hunk 1]
```diff
@@ -226,7 +226,7 @@ def claim(self, statement_text: str):
         publication.provenance.add((rdflib.URIRef(profile.get_orcid_id()),
                                     namespaces.HYCL.claims,
                                     rdflib.URIRef(DUMMY_NANOPUB_URI + '#mystatement')))
-        self.publish(publication)
+        return self.publish(publication)
```
Comment (svenvanderburg) [inline nanopub/client.py:229]: This is also useful!

Comment (raar1) [reply nanopub/client.py:229]: Agreed.

Comment (raar1) [reply nanopub/client.py:229]: Needed for consistency

**Review by raar1 — APPROVED** [review]
Simple PR, useful changes

**Pull request merged**